docs/proxy.md

```diff
@@ -52,13 +52,11 @@ Sessions are available for [datacenter]({{@link proxy/datacenter_proxy.md#sessio
 
 ### [](#session-persistence) Session persistence
 
-You can persist your sessions (use the same IP address) by setting the `session` parameter in the `username` [field]({{@link proxy/connection_settings.md#username-parameters}). This assigns a single IP address to a **session ID** after you make the first request.
+You can persist your sessions (use the same IP address) by setting the `session` parameter in the `username` [field]({{@link proxy/connection_settings.md#username-parameters}}). This assigns a single IP address to a **session ID** after you make the first request.
 
 **Session IDs represent IP addresses. Therefore, you can manage the IP addresses you use by managing sessions.** In cases where you need to keep the same session (e.g. when you need to log in to a website), it is best to keep the same proxy. By assigning an IP address to a **session ID**, you can use that IP for every request you make.
 
-For datacenter proxies, a session persists for **24 hours** ([more info]({{@link proxy/datacenter_proxy.md#session-persistence}})).
-
-For residential proxies, a session persists for **1 minute** ([more info]({{@link proxy/residential_proxy.md#session-persistence}})).
+For datacenter proxies, a session persists for **24 hours** ([more info]({{@link proxy/datacenter_proxy.md#session-persistence}})). For residential proxies, it persists for **1 minute** ([more info]({{@link proxy/residential_proxy.md#session-persistence}})).
```

Comment:
Would be good to say that by using the session you refresh the session expiry timer.